Sources/MuxUploadSDK/InternalUtilities/ChunkedFile.swift

```diff
@@ -39,11 +39,21 @@ class ChunkedFile {
             return Result.failure(ChunkedFileError.fileHandle(error))
         }
     }
+
+    func fetchFileSize(
+        fileURL: URL
+    ) throws -> UInt64 {
+        guard let fileSize = try FileManager.default.attributesOfItem(atPath: fileURL.path)[FileAttributeKey.size] as? UInt64 else {
+            throw ChunkedFileError.invalidState("Cannot retrieve file size")
+        }
+
+        return fileSize
+    }
     
     /// Opens the internal file ahead of time. Calling this is optional, but it's available
     /// Calling this multiple times (on the same thread) will have no effect unless you also ``close`` it
     /// Throws if the file couldn't be opened
-    public func openFile(fileURL: URL) throws {
+    func openFile(fileURL: URL) throws {
```

Comment:
Am assuming this wasn't intended to originally be public

Reply:
(Given the `ChunkedFile` isn't public to begin with)

Reply:
nitpick: Given the description, I think the intention was at one point to make this public. If that's the case, may want to revert since it's not a necessary change for this PR. If not, update the comment appropriately.

Reply:
It's supposed to be internal. Making the methods `public` doesn't expose them since the class is `internal` but then again there's no reason to make them `public`. I'm behind this change

Reply:
Changed in #51